packages/cli/src/commands/experimental/templates/streamingSsr/tsconfig.json.template

```diff
@@ -0,0 +1,54 @@
+{
+  "compilerOptions": {
+    "noEmit": true,
+    "allowJs": true,
+    "esModuleInterop": true,
+    "target": "esnext",
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext",
+    "skipLibCheck": false,
+    "rootDirs": [
+      "./src",
+      "../.redwood/types/mirror/web/src",
+      "../api/src",
+      "../.redwood/types/mirror/api/src"
+    ],
+    "paths": {
+      "src/*": [
+        "./src/*",
+        "../.redwood/types/mirror/web/src/*",
+        "../api/src/*",
+        "../.redwood/types/mirror/api/src/*"
+      ],
+      "$api/*": [
```

Comment:
Can you please try to keep this file as close as possible to the template in CRWA? So that it's easier for users to see what changes the setup command actually did by just running a `git diff`. Thanks!

Reply:
![image](https://github.com/redwoodjs/redwood/assets/30793/aaeb2575-76ab-4406-9c6b-738b591abc99)

Hmm...

Why did you have to do this?

Reply:
Sure thing. The CRWA template isn't formatted with prettier (which is why you see a big diff. I used prettier on all of the ones I changed. 

Would you like me to prettify crwa or change all the others to the unformatted version?